PHP/PHP Source.sublime-syntax

```diff
@@ -2716,7 +2718,7 @@ contexts:
         [-+]? (?: [ 0] | \\'. )?    # flags
         \d*                         # width
         (?: (\.) \d* )?             # precision
-        [bcdeEfFgGhHosuxX]          # specifier
+        [aAbBCcdDeEFfgGhHIjklmMnopPrRSsTtuUVwWXxyYzZ] # specifier
```

Comment:
`C`, `F`, `S`, `T`, and `X` do not follow the lower-case-first pattern. The whole set can be simplified to `[A-IMPS-Za-hj-pr-z]`, but that might be hard to read. I don't know.

Alternatively, make variables and put them into both placeholder regexes.

```yaml
  format_printf: '[bcdeEfFgGhHosuxX]'
  format_strftime: '[somethingNEW]'
  format_specifier: (?:{{format_printf}}|{{format_strftime}})
```

Reply:
Variables would cause various duplicates. Not sure if ST would optimize them away. Not sure if changes are to be expected so adding vars is justified.

Reply:
I stayed with explicit list of chars to be able to compare them one-by-one with specs in case they change in future.

Reply:
... too lazy to sort test cases.